corehq/apps/smsforms/models.py

```diff
@@ -431,6 +432,7 @@ def get_channel_for_contact(contact_id, phone_number):
     backend_id = None
     phone_number_record = PhoneNumber.get_phone_number_for_owner(contact_id, phone_number)
     if phone_number_record:
+        phone_number = phone_number_record.phone_number  # prefer this number since it has leniency applied
```

Comment:
To confirm my understanding, this won't actually matter because leniency will be applied again when getting the key right?

Reply:
Yes, that's my understanding.

Reply:
that is true but I still think this is worth keeping